timesketch/api/v1/resources/event.py

```diff
@@ -921,6 +923,15 @@ def post(self, sketch_id: int):
                     if "__ts_star" in form.annotation.data:
                         search_node_label = "__ts_star"
                     current_search_node.add_label(search_node_label)
+            
+                # Adding facts to conclusions
+                if current_search_node and "__ts_fact" in form.annotation.data:
+                    if hasattr(current_search_node, 'investigativequestion') and current_search_node.investigativequestion:
+                        if hasattr(current_search_node.investigativequestion, 'conclusions') and current_search_node.investigativequestion.conclusions:
```

Comment:
How are you going to handle situations where the conclusion object does not exist yet?